### ISSUE TITLE
error: version control conflict marker in FieldNodeDefs.h

### ISSUE THREAD

**Issue by ilovezfs** [jimbraun/XCDF#82]
https://github.com/jimbraun/XCDF/blob/master/include/xcdf/utility/FieldNodeDefs.h#L52-L55

```
    // Knowing size limits is up to the user
    T operator[](unsigned index) const {
      const T& datum = field_[index];
      return datum;
<<<<<<< HEAD
=======
      //return field_[index];
>>>>>>> 8d9d6fcde1000c8369a52eed108c8461af0de450
    }
    unsigned GetSize() const {return field_.GetSize();}

    const std::string& GetName() const {return field_.GetName();}
```

So the 3.00.01 release will not build.

**Comment by jimbraun** [comment]
Fixed with 3.00.02 release

**Comment by ilovezfs** [comment]
Thanks!
